src/ostorlab/agent/schema/agent_group_schema.json

```diff
@@ -146,6 +138,10 @@
           "depth_processing_limit": {
             "description": "[Optional] - Count of how deep the processing of a message can go through the agents, if no value is specified, means no limit.",
             "type": "number"
+          },
+          "accepted_agents": {
+            "description": "[Optional] - Array of strings of the accepted agents, if no value is specified, means all agents are accepted.",
+            "$ref": "#/CustomTypes/ArrayOfStrings"
```

Comment:
Are the agents in the form of key `agent/ostorlab/nmap` or just name `nmap`.

Reply:
just the name